kakao/KakaoLoginIOS/KakaoLoginIOS/ViewController.swift

```diff
@@ -13,34 +21,22 @@ import SDWebImage
 class ViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
-
-    }
-    
-    override func viewDidAppear(_ animated: Bool) {
-        super.viewDidAppear(animated)
         if Auth.auth().currentUser == nil {
             if KOSession.shared().isOpen() {
                 requestFirebaseJwt(accessToken: KOSession.shared().accessToken)
             }
         } else {
-            self .performSegue(withIdentifier: "loginSegue", sender: self)
-            
+            self.performSegue(withIdentifier: "loginSegue", sender: self)
```

Comment:
This has the same caveat of performing segues in `viewDidLoad` as my last comment.